public/css/gitgraph.css

```diff
@@ -10,6 +10,6 @@ em {font-style:normal;}
 #git-graph-container li a {color:#000000;}
 #git-graph-container li a:hover {text-decoration:underline;}
 #git-graph-container li a em {color:#BB0000;border-bottom:1px dotted #BBBBBB;text-decoration:none;font-style:normal;}
-#rev-container {width:80%}
-#rev-list {margin:0;padding:0 5px 0 0;width:80%}
+#rev-container {width:100%}
+#rev-list {margin:0;padding:0 5px 0 0;width:100%}
```

Comment:
did you try this locally. On my test, with the repo from the issue, it looks better using 100%: 

![screenshot from 2017-01-09 21-54-28](https://cloud.githubusercontent.com/assets/3092520/21783157/75a65438-d6b6-11e6-9cdf-9ded0782dafe.png)


95%:

![screenshot from 2017-01-09 21-54-22](https://cloud.githubusercontent.com/assets/3092520/21783158/75a6d16a-d6b6-11e6-9fc3-224a13b6dddb.png)

(this was on chrome, around 900px wide)

what do you think?